python/ray/setup-dev.py

```diff
@@ -60,6 +60,7 @@ def do_link(package, force=False, local_path=""):
     do_link("experimental", force=args.yes)
     do_link("util", force=args.yes)
     do_link("dashboard", force=args.yes)
+    do_link("new_dashboard", force=args.yes)
```

Comment:
shouldn't this be in the other PR?

Reply:
I cherry-picked the commit to both for testing purposes.

Reply:
I cherry-picked to both for testing.